src/main/java/com/chessmates/repository/MockGameRepository.groovy

```diff
@@ -26,7 +26,7 @@ class MockGameRepository implements GameRepository {
     }
 
     @Override
-    List findAll() {
+    def findAll() {
```

Comment:
Why do we change it to def? Maybe having List is useful as it lets use use some type chacking? What do you think?